src/utils/CustomPropTypes.js

```diff
@@ -38,6 +54,10 @@ const CustomPropTypes = {
   all
 };
 
+function errMsg(props, propName, componentName) {
+  return `Invalid prop '${propName}' of value '${props[propName]}' supplied to '${componentName}'`;
```

Comment:
Maybe i am just being dense but it doesn't look like this actually deals with the 4 argument, that is passed to this most of the time? Should there also be a `message` prop or something that is concated to the end of the initial one?